### PR TITLE
Pract.04: Add 01-03 | Danail Todorov

### PR DESCRIPTION
**Pull request by Danail3**
Danail Todorov
6 0MI0600241

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task01.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+
+int Sum(int a, int b)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task01.cpp:4]: Use camelCase for naming your functions

File: SI/Sem.04/Pract.04/Task02.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+bool isEven(int num)
+{
+	bool isEven;
+	if (num % 2 == 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task02.cpp:7]: num % 2 is a boolean expression, so it returns a boolean value that you can directly return

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+using namespace std;
+
+bool isPositive(int num)
+{
+	bool isPositive;
+	if (num >= 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:7]: num >= 0 is a boolean expression, so it returns a boolean value that you can directly return

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	{
+		isPositive = false;
+	}
+	return isPositive;
+}
+
+bool isNegative(int num)
+{
+	bool isNegative;
+	if (num < 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:21]: num < 0 is a boolean expression, so it returns a boolean value that you can directly return

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Some refactoring needed

**Comment by Justsvetoslavov** [comment]
Closing due to merged solutions.
Address comments and contact us if you have questions.

**Pull request closed without merging**